src/test/java/login/LoginTests.java

```diff
@@ -0,0 +1,43 @@
+package login;
```

Comment:
delete this file from your PR
Your taste is registration not Login..

Reply:
Done

Reply:
Done